app/src/main/java/org/jellyfin/androidtv/data/service/BackgroundService.kt

```diff
@@ -181,9 +182,15 @@ class BackgroundService(
 						.centerCrop()
 						.submit()
 				}
-				.map { future -> async { future.get() } }
+				.map { future -> async {
+					try {
+						future.get()
+					} catch (ex: Exception) {
+						null
```

Comment:
Should be a more specific exception to catch  (GlideException) and report it with Timber.e